component.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: Component
+spec:
+  type: scheduler
```

Comment:
How do you intend to use this for replicated components?

How do you intend presence monitoring to work (i.e., detecting that a component should exist but hasn't yet registered)?